shared/android/app/src/main/AndroidManifest.xml

```diff
@@ -57,6 +57,13 @@
           <category android:name="android.intent.category.BROWSABLE" />
           <data android:scheme="keybase" />
         </intent-filter>
+        <intent-filter>
+          <action android:name="android.intent.action.VIEW" />
+          <category android:name="android.intent.category.DEFAULT" />
+          <category android:name="android.intent.category.BROWSABLE" />
+          <data android:scheme="http" android:host="keybase.io" android:pathPrefix="/" />
```

Comment:
we need to discuss

Reply:
I have literally no idea how we should approach this thing - we mostly just need a regexp so that this only happens on usernames _and_ I believe it should work exactly the same way as on iOS (which I haven't looked into, all I know is that it opens the app there). I can add the link verification logic on the server if we go forward with this change.

Reply:
I found that some people are using https://github.com/airbnb/DeepLinkDispatch for more complicated matching. I limited it to the 2 special routes instead.